packages/flutter/example/macos/Flutter/ephemeral/Flutter-Generated.xcconfig

```diff
@@ -1,6 +1,7 @@
 // This is a generated file; do not edit or check into version control.
-FLUTTER_ROOT=C:\flutter
-FLUTTER_APPLICATION_PATH=D:\GithubProjects\parse\Parse-SDK-Flutter\packages\flutter\example
+FLUTTER_ROOT=/Users/rodrigo/flutter
```

Comment:
what are these local paths? should these files be in gitignore?

Reply:
Yes

Reply:
New created flutter projects have the following in the .gitignore of the macos directory:
```
# Flutter-related
**/Flutter/ephemeral/
**/Pods/

# Xcode-related
**/dgph
**/xcuserdata/
```